python/lsst/obs/lsstCam/lsstCamMapper.py

```
@@ -181,7 +182,7 @@ def assemble_raw(dataId, componentInfo, cls):
 
     try:
         exposure = _lsstCamMapper.std_raw(exposure, dataId)
-    except:
+    except:  # noqa E722
```

Is there a reason for the bare except?  Surely it should at least be `except Exception`, right?

---

You should never mask this. It needs to be fixed to decide between `Exception` and `BaseException`.

---

I don't understand this.  I want to catch any problem, so I can change this to catch `(Exception, BaseException)` if you like, but I think that the `noqa E722` is clearer.

---

Python 3 changed everything. Bare except is now deprecated because it's not entirely clear what you mean. If you mean *everything* then you say you mean everything by using `BaseException`. If you don't mean catching Ctrl-C and `sys.exit` then you want to use `Exception` instead.